app/models/package_manager/maven.rb

```diff
@@ -223,7 +223,7 @@ def self.from_name(name)
         new(*name.split(':', 2))
       end
 
-      def initialize(group_id, artifact_id)
+      def initialize(group_id = nil, artifact_id = nil)
```

Comment:
Won't this then explode later when trying to concat `nil` into strings?

Reply:
Potentially. I'm not sure how to get bad data into my environment at the moment to test. I can come back to this tomorrow and try and give it more thought.

Reply:
Ok I was able to get a local example into elasticsearch and have it come back in the search results. The `nil` default seems to work fine in the search scenario and since it is only used in string interpolations, I don't think it will be an issue within the `MavenUrl` class. I don't know if it will have any implications anywhere else in the app. I could also change the default to an empty string to at least have the correct type for the variables.